components/collection/activity/events/Events.vue

```diff
@@ -118,26 +115,18 @@ const getEventAddresses = (event): string[] => {
 
 const eventsAddresses = computed(() => {
   const addresses = props.events.map(getEventAddresses).flat()
-
-  return [...new Set([...addresses])].map(getIdentityId)
+  return [...new Set([...addresses])]
 })
 
-const { data: identities, refetch: getIdentities } = useGraphql({
-  clientName,
-  queryName: 'identities',
-  disabled: computed(() => true),
-  variables: { where: {} },
-})
+const { data: profiles, refresh: refreshProfiles } = useAsyncData(
+  () => fetchProfilesByIds(eventsAddresses.value),
+  { immediate: true },
+)
```

Comment:
consider giving it a key so it can be cached
and prevent calls with empty ids

![image](https://github.com/kodadot/nft-gallery/assets/22791238/0953d825-f422-4e0a-9404-f92890896d57)